docs/reference/commandline/cli.md

```diff
@@ -78,8 +78,9 @@ For example:
 Instructs Docker to use the configuration files in your `~/testconfigs/`
 directory when running the `ps` command.
 
-> **Note**: If you run docker commands with `sudo`, Docker first looks for a configuration
-> file in `/root/.docker/`, before looking in `~/.docker/` for the user that did the sudo call.
+> **Note**: If you run docker commands with `sudo`, Docker looks for a configuration
+> file in `/root/.docker/` by default. In order to make it look for a configuration
+> file for the user who did the call, you have to use `sudo -u <username>`.
```

Comment:
This isn't accurate. If I'm reading the code right, it's going to look for a configuration file in `$HOME/.docker`. This doesn't necessarily mean `/root` if `sudo` is used. `sudo` has a `-E` option to preserve environment variables, including `$HOME`.